tests/e2e/sample-applications/virtual-machines/cirros-test/cirros-test-immediate.yaml

```diff
@@ -0,0 +1,48 @@
+apiVersion: v1
+kind: List
+items:
+  - apiVersion: kubevirt.io/v1
+    kind: VirtualMachine
+    metadata:
+      name: cirros-test
+      namespace: cirros-test
+    spec:
+      dataVolumeTemplates:
+        - apiVersion: cdi.kubevirt.io/v1beta1
+          kind: DataVolume
+          metadata:
+            name: cirros-test-disk
+          spec:
+            sourceRef:
+              kind: DataSource
+              name: cirros
+              namespace: openshift-virtualization-os-images
+            storage:
+              resources:
+                requests:
+                  storage: 128Mi
+              storageClassName: test-gp3-csi-immediate
```

Comment:
only difference between `tests/e2e/sample-applications/virtual-machines/cirros-test/cirros-test.yaml` and this file is this line, right?

would be better to add logic in code to add this to installed VM?

Reply:
Yes, that would be fine to add at VM installation time. There would just need to be a new VmBackupRestoreCase field to specify the volumeMode.

Reply:
On second thought, I don't think this is configurable after the VM is created. It could be another use case for templating, or maybe some kind of hook inside InstallApplication? I will probably keep this file for this pull request, and try some other options out later.